src/lib/services/data-converter.ts

```diff
@@ -10,6 +10,35 @@ interface WebSocketResponse {
   content: string;
   requestId: string;
 }
+
+export async function convertSinglePayloadDataConverter(
```

Comment:
You can probably just name this `convertPayload`. TypeScript ensures it’s a single payload and it’s coming from `data-converter.ts`.

Reply:
Yeah, the other function should be deleted but I was keeping it around for reference.